content/en/glossary/terms/test_regression.md

```diff
@@ -6,6 +6,7 @@ core_product:
 related_terms:
   - performance regression
 ---
-In Datadog CI Test Visibility, a test run is marked as a regression when its duration is both five times the mean and greater than the max duration for the same test in the default branch. A benchmark test run is marked as a regression when its duration is five times the standard deviation above the mean for the same test in the default branch. 
+In Datadog CI Test Visibility, a test run is marked as a regression when its duration is both five times the mean and greater than the max duration for the same test in the default branch. A benchmark test run (`@test.type:benchmark`) is marked as a regression when its duration is five times the standard deviation above the mean for the same test in the default branch.
 
-A benchmark test has `@test.type:benchmark`. The mean and the max of the default branch is calculated over the last week of test runs. For more information, <a href="/tests/search/#test-regressions">see the documentation</a>.
\ No newline at end of file
+The mean and maximum durations of the default branch are calculated based on the test runs from the past week. For the algorithm to consider a test run, there must be a minimum of 100 test runs on the default branch. For benchmark test runs, the minimum number of test runs is 10.
+For more information, <a href="/tests/search/?tab=tests#test-regressions">see the documentation</a>.
```

Comment:
```suggestion
For more information, <a href="/tests/search/#test-regressions">see the documentation</a>.
```
The other link will work, but we should remove the tab param in case it causes confusion in the future.